Lib/test/test_syntax.py

```diff
@@ -461,22 +461,22 @@
 >>> def foo(a,*b=3,c):
 ...    pass
 Traceback (most recent call last):
-SyntaxError: var-positional parameter cannot have default value
+SyntaxError: var-positional parameter's default value cannot be changed because it has the immutable default value ()
```

Comment:
I don't think this is clearer. "var-positional parameter cannot have default value" is oddly worded but I think equal or better to this proposal.